packages/api/src/graphql/article.ts

```diff
@@ -119,10 +119,10 @@ export const GraphQLArticleRevision = new GraphQLObjectType<ArticleRevision, Con
     hideAuthor: {type: GraphQLNonNull(GraphQLBoolean)},
 
     preTitle: {type: GraphQLString},
-    title: {type: GraphQLNonNull(GraphQLString)},
+    title: {type: GraphQLString},
```

Comment:
Why nullable now/ missed that.

Reply:
A lot of fields were filled with empty string by default. Made them nullable now.